src/components/Column/Column.js

```diff
@@ -3,10 +3,10 @@ import { useSelector } from 'react-redux';
 import styles from './Column.module.scss';
 import Card from '../Card/Card';
 import CardForm from '../CardForm/CardForm';
-import { getFilteredCards } from '../../redux/store'; // Upewnij się, że ta ścieżka jest poprawna
+import { getFilteredCards } from '../../redux/cardRedux'; // Upewnij się, że ścieżka do importu jest poprawna
 
 const Column = ({ id, title, icon }) => {
-    const cards = useSelector(state => getFilteredCards(state, id));
+    const cards = useSelector(state => getFilteredCards(state, id, ''));
```

Comment:
Pusty string jako trzeci argument jest niepotrzebny. Zauważ, że przy definicji selektora `getFilteredCards` deklarujesz tylko dwa argumenty.
```suggestion
    const cards = useSelector(state => getFilteredCards(state, id));
```